todo_app/data/trello_items.py

```diff
@@ -0,0 +1,58 @@
+import os
+import requests
+
+from todo_app.data.item import Item
+
+def get_items():
+    reqUrl = "https://api.trello.com/1/boards/65afd1d3a662c6c7228ce261/lists"
```

Comment:
Even though it's not a secret, we should still extract this id variable so that we can vary the environments later (e.g. we don't want our test site loading the same data as production!)
```suggestion
    reqUrl = f"https://api.trello.com/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"
```